NYPLAudiobookToolkit/Network/AudiobookNetworkService.swift

```diff
@@ -207,6 +274,28 @@ extension DefaultAudiobookNetworkService: DownloadTaskDelegate {
         }
     }
     
+    /// - Important: Must be called on the main thread.
```

Comment:
since this implementation does nothing on the main thread, I would move that responsibility downstream , i.e. to whoever implements `audiobookNetworkService(_:didTimeoutFor:networkStatus:)`. 

Basically there's no real difference from this point of view between `notifyDelegatesOfTimeoutFor(_:networkStatus:)` and `notifyDelegatesOfDownloadExceededTimeLimitFor(_:elapsedTime:networkStatus:)`, so I think they should all have the same main thread requirements to avoid confusion. 

I would also move the `For` inside the parameter list per swift conventions. (`notifyDelegatesOfTimeout(for spineElement...)`)